libs/payments/eligibility/src/lib/eligibility.manager.spec.ts

```diff
@@ -30,30 +30,30 @@ import {
 
 describe('EligibilityManager', () => {
   let manager: EligibilityManager;
-  let mockContentfulManager: ContentfulManager;
-  let mockOfferingResult: EligibilityContentByOfferingResultUtil;
-  let mockResult: EligibilityContentByPlanIdsResultUtil;
-  let mockStripeManager: StripeManager;
+  let contentfulManager: ContentfulManager;
+  let offeringResult: EligibilityContentByOfferingResultUtil;
+  let eligibilityResult: EligibilityContentByPlanIdsResultUtil;
+  let stripeManager: StripeManager;
 
```

Comment:
Just a general question for further discussion, why are we renaming these values?

My thinking is that I quite liked having "mock" at  the start of the name, so that it's easy to identify what is a mock and what isn't.

Reply:
Great question. This was after reviewing with Julian yesterday and he answered some of it in comments below. I do have to go through and revise. For the ones that aren't mocks, we shouldn't begin with mock and for the ones that are, we should. Now, I changed some that didn't need changing and will go ahead and revert those, but throughout our code, there are cases when mock is used and shouldn't be. 👍